src/core/src/pass/sdpa_to_paged_attention.cpp

```diff
@@ -137,6 +144,15 @@ bool ov::pass::SDPAToPagedAttention::run_on_model(const std::shared_ptr<ov::Mode
         model->remove_result(result);
     }
 
+    if (m_use_cache_eviction) {
+        block_indices->set_partial_shape(PartialShape{layer_index}); // The shape of block_indices should correspond to the number of PA nodes
+
+        auto scores_concat = std::make_shared<v0::Concat>(OutputVector{scores_outputs}, 0);
+        scores_concat->output(0).set_tensor_ptr(std::make_shared<ov::descriptor::Tensor>(scores_concat->get_element_type(), PartialShape{layer_index}));
```

Comment:
Here I use Concat to concatenate PAs' nodes outputs and I set the output shape explicitly (to the number of PA). However, it doesn't really affect the final result's shape as it is dynamic. Do you think this is correct to do and if I need it at all?

Reply:
Btw, do you think the Concat way is valid here?